docs/proposals/20220603-kubernetes-vendor.md

```diff
@@ -0,0 +1,235 @@
+# Multi-runtime Support
+
+Authors:
+- [Ce Gao](https://github.com/gaocegege)
+
+## Summary
+
+envd builds and runs the environments with Docker. This proposal is to support Kubernetes as a runtime option. After this feature, envd can build the environment with Docker and run on Kubernetes.
+
+## Motivation
+
+Docker runtime only works for individual developers. There are many data scientists developing models on Kubernetes. In the current design, they may need to run `envd build`, then push the image to a OCI image registry manually. The image can be used after a deployment or a pod is created in Kubernetes.
+
+We can provide native support for ease of use.
+
+## Goals
+
+- Build the envd environments with the buildkitd pod on Kubernetes
```

Comment:
Is `buildkitd` a typo?

Reply:
I forgot to add the terms section to illustrate it. 

Will add it soon. As for the question, buildkitd is the daemon process of buildkit.